### DISCUSSION TITLE
Paginated tag-based archives

### DISCUSSION THREAD

**Discussion by khawkins98** [11ty/eleventy#1993] [Q&A]
Stumped on how to elegantly make something like `site.com/archive/tag-name/page-3` 

I'm familiar with 11ty's pagination methods and this: https://www.11ty.dev/docs/quicktips/tag-pages/

Those methods do allow you to loop through posts/an object to make:

- `site.com/archive/tag-name-1`
- `site.com/archive/tag-name-2`
- `site.com/archive/tag-name-3`

However in my case we want to have, say, 9 posts per archive page. So a secondary loop is needed. 

- `site.com/archive/tag-name-1/page-1`
- `site.com/archive/tag-name-1/page-2`
- `site.com/archive/tag-name-1/page-3`
- `site.com/archive/tag-name-2/page-1`
- `site.com/archive/tag-name-2/page-2`
- `site.com/archive/tag-name-2/page-3`
- and so on

With `pagination` sensibly only supporting one level of loops, is there a good way to achieve what I want? Surely someone else has tried this, but Google gives me nothing.

I've considered somehow dynamically generating a `.njk` post per tag name, but it feels awfully hacky and inelegant.

**Comment by pdehaan** [comment]
Not sure if there is an easy way to do dual pagination, but these threads might have some hot tips(tm):
- #1120
- #332
- #793

@zachleat said in https://github.com/11ty/eleventy/issues/776#issuecomment-558637460:

> Ah this is the double-layer pagination problem. There is a way to do this but it isn’t super friendly right now.
> See the example I posted here: #332
> **Also upvote #461 if you’d like to see a nicer way integrated into core**

**Reply by khawkins98** [reply]
Thanks for the info and links — #332 duly upvoted.
